pkg/api/endpoints/util.go

```diff
@@ -28,6 +28,16 @@ import (
 	hashutil "k8s.io/kubernetes/pkg/util/hash"
 )
 
+const (
+	// Its value is the json representation of map[string(IP)][HostRecord]
+	// example: '{"10.245.1.6":{"HostName":"my-webserver"}}'
+	PodHostNamesAnnotation = "endpoints.net.beta.kubernetes.io/hostnames-map"
```

Comment:
I think its ok to leave it as is. The map is auto generated by json serialization of the go map.